src/arviz_plots/backend/none/__init__.py

```diff
@@ -398,6 +398,36 @@ def hline(y, target, *, color=unset, alpha=unset, width=unset, linestyle=unset,
     return artist_element
 
 
+def vspan(xmin, xmax, target, color=unset, alpha=unset, **artist_kws):
```

Comment:
```suggestion
def vspan(xmin, xmax, target, *, color=unset, alpha=unset, **artist_kws):
```

it shouldn't really change anything but this will match the other functions in the module and the equivalent functions in the other backends